ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewState.kt

```diff
@@ -11,12 +12,12 @@ internal sealed class PaywallViewState {
     data class Error(val errorMessage: String) : PaywallViewState()
     data class Loaded(
         val templateConfiguration: TemplateConfiguration,
-        val selectedPackage: TemplateConfiguration.PackageInfo,
+        val selectedPackage: MutableState<TemplateConfiguration.PackageInfo>,
```

Comment:
Hmm I was doubting whether exposing a `MutableState` to the View model and the data layer was a good idea, but seems to be accepted from what I've seen.

Reply:
And looks like it's the only way to be able to derive animations from it. I tried making it private and exposing only the value, but that doesn't work.